HM17/tests/test_site.py

```diff
@@ -0,0 +1,207 @@
+
+
+@pytest.mark.main
+@pytest.mark.regression
+def test_go_to_pricing(open_main_page):
+    main_page = open_main_page
+    price = main_page.click_buy_now_button()
+    assert price.is_pricing_displayed(), 'Not in pricing page'
+
+
+@pytest.mark.header
+def test_go_to_pricing_header(open_main_page):
+    main_page = open_main_page
+    price_head = main_page.click_x_in_banner().click_pricing_head_button()
+    assert price_head.is_pricing_displayed(), 'Not in pricing page'
+
+
+@pytest.mark.header
+def test_open_dropdown_products(open_main_page):
+    main_page = open_main_page
+    products_head = main_page.click_button_products()
+    assert products_head.is_open_dropdowm_header(), 'Not open dropdown products'
+
+
+@pytest.mark.header
+def test_open_product_spider_in_dropdown_header(open_main_page):
+    main_page = open_main_page
+    spider_head = main_page.click_button_products().click_in_dropdown_products_spider_head()
+    assert spider_head.is_spider_page_displayed(), 'Not open dropdown spider'
+
+
+@pytest.mark.header
+def test_open_product_checker_in_dropdown_header(open_main_page):
+    main_page = open_main_page
+    checker_head = main_page.click_button_products().click_in_dropdown_products_checker_head()
+    assert checker_head.is_checker_page_displayed(), 'Not open dropdown checker'
+
+
+@pytest.mark.header
+def test_open_pop_up_in_header(open_main_page):
+    main_page = open_main_page
+    popup = main_page.click_free_download_button_in_header()
+    assert popup.is_popup_register_displayed(), 'Popup not opened in header'
+
+
+@pytest.mark.header
+def test_open_pop_up_login_in_header(open_main_page):
+    main_page = open_main_page
+    popup_log = main_page.click_login_in_header()
+    assert popup_log.is_popup_login_displayed(), 'Popup login not opened in header'
+
+
+@pytest.mark.pricing
+def test_open_pricing_page(open_pricing_page):
+    pricing_page = open_pricing_page
+    assert pricing_page.is_pricing_displayed(), 'Pricing page not opened'
+
+
+@pytest.mark.pricing
+def test_open_pricing_checker(open_pricing_page):
+    pricing_page = open_pricing_page
+    checker_tab = pricing_page.click_checker_tab_button()
+    assert checker_tab.is_checker_tab_displayed(), 'Not open checker'
+
+
+@pytest.mark.pricing
+def test_change_to_lifetime_spider(open_pricing_page):
+    pricing_page = open_pricing_page
+    lifetime_spider = pricing_page.click_lifetime_radiobutton()
+    assert lifetime_spider.is_lifetime_displayed(), 'Not in lifetime'
+
+# def test_click_logo_in_header(open_login_page):
+#     global main_page_go
+#     login_page = open_login_page
+#     try:
+#         main_page_go = login_page.click_logo_header()     не получилось понять как обойти алерт
+#     except UnexpectedAlertPresentException:
+#         alert = login_page.driver.switch_to.alert
+#         alert.accept()
+#     assert main_page_go.is_h1_main_page_displayed(), 'H1 not displayed'
```

Comment:
you should separate TC into suites. Also, all verification is on displaying only. What if an element is displayed but has the wrong text for example? You should check different data